_includes/header/header-en.html

```diff
@@ -1,156 +1,316 @@

-                          </li>

-                          <li><a href="/{{ page.lang }}/advanced/security-updates.html">Security updates</a>

-                          </li>

-                          <li><a href="/{{ page.lang }}/advanced/best-practice-security.html">Security best practices</a>

-                          </li>

-                          <li><a href="/{{ page.lang }}/advanced/best-practice-performance.html">Performance best practices</a>

-                          </li>

-                      </ul>

-                  </li>

-              </ul>

-          </li>

-          <li>

-              <ul id="resources-menu" class="menu">

-                  <li><a href="/{{ page.lang }}/resources/glossary.html"{% if page.menu == 'resources' %} class="active"{% endif %}>Resources</a>

-                      <ul>

-                          <li>

-                            <a href="/{{ page.lang }}/resources/community.html">Community</a>

-                          </li>

-                          <li>

-                            <a href="/{{ page.lang }}/resources/glossary.html">Glossary</a>

-                          </li>

-                          <li>

-                            <a href="/{{ page.lang }}/resources/middleware.html">Middleware</a>

-                          </li>

-                          <li>

-                            <a href="/{{ page.lang }}/resources/utils.html">Utility modules</a>

-                          </li>

-                          <li>

-                            <a href="/{{ page.lang }}/resources/frameworks.html">Frameworks</a>

-                          </li>

-                          <li>

-                            <a href="/{{ page.lang }}/resources/books-blogs.html">Books and blogs</a>

-                          </li>

-                          <li>

-                            <a href="/{{ page.lang }}/resources/companies-using-express.html">Companies using Express</a>

-                          </li>

-                          <li>

-                            <a href="/{{ page.lang }}/resources/contributing.html">Contributing to Express</a>

-                          </li>

-                          <li>

-                            <a href="/{{ page.lang }}/changelog/4x.html">Release Change Log</a>

-                          </li>

-                      </ul>

-                  </li>

-              </ul>

-          </li>

-          <!--

-          <li>

-              <ul id="changelog-menu" class="menu">

-                  <li><a href="/{{ page.lang }}/changelog/4x.html"{% if page.menu == 'changelog' %} class="active"{% endif %}>Changelog</a>

-                      <ul>

-                          <li><a href="/{{ page.lang }}/changelog/4x.html">4.x</a></li>

-                      </ul>

-                  </li>

-              </ul>

-          </li>

-        -->

-      </ul>

-  </div>

-</header>

+<header>
+

```

Comment:
Not sure what is going on in this change, adding a blank line between every single original line?

Reply:
Hm, that's a good question - will look into this too